source/includes/gcp/_routers.md

```diff
@@ -126,3 +126,122 @@ Retrieve a cloud router in a given [environment](#administration-environments).
 | `bgp`<br/>_Object_               | BGP information specific to this router.                                                                                                     |
 | `bgpPeers`<br/>_Object_          | BGP information that is configured into the routing stack to establish BGP peering. _(Includes: `Peer ASN`, `Interface name`, `IP address`)_ |
 | `interfaces`<br/>_Array[Object]_ | Router interfaces. Each interface contains either one linked resource or IP address and IP address range.                                    |
+
+<!-------------------- CREATE A ROUTER -------------------->
+
+#### Create a router
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/routers"
+```
+
+> Request body example:
+
+```json
+{
+  "name": "cmc-c-router",
+  "shortNetwork": "cmc-vpc",
+  "shortRegion": "us-central1",
+  "bgp": {
+    "asn": 65510
+  }
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/routers</code>
+
+Create a new router
+
+| Required                    | &nbsp;                                                                                                                          |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| `name`<br/>_string_         | The display name of the router.                                                                                                 |
+| `shortRegion`<br/>_string_  | A short version of the region name.                                                                                             |
+| `shortNetwork`<br/>_string_ | A short version of the network name to which this router is attached to.                                                        |
```

Comment:
There are two "to"s. It can be `A short version of the network name to which this router is attached.` or `A short version of the network name this router is attached to.`

Reply:
Rephrased in https://github.com/cloudops/cloudmc-api-docs/pull/155/commits/e85e04a860ab69deb275abff2dacc6d47d56eeb0